model_info/viwiki.articletopic.md

```diff
@@ -15,947 +15,947 @@ Model Information:
+		Culture.Visual arts.Visual arts*             0.972
+		Geography.Geographical                       0.965
+		Geography.Regions.Africa.Africa*             0.933
+		Geography.Regions.Africa.Central Africa      0.956
+		Geography.Regions.Africa.Eastern Africa      0.903
+		Geography.Regions.Africa.Northern Africa     0.939
+		Geography.Regions.Africa.Southern Africa     0.953
+		Geography.Regions.Africa.Western Africa      0.963
+		Geography.Regions.Americas.Central America   0.915
+		Geography.Regions.Americas.North America     0.924
+		Geography.Regions.Americas.South America     0.933
+		Geography.Regions.Asia.Asia*                 0.912
+		Geography.Regions.Asia.Central Asia          0.949
+		Geography.Regions.Asia.East Asia             0.947
+		Geography.Regions.Asia.North Asia            0.937
+		Geography.Regions.Asia.South Asia            0.93
+		Geography.Regions.Asia.Southeast Asia        0.914
+		Geography.Regions.Asia.West Asia             0.944
+		Geography.Regions.Europe.Eastern Europe      0.932
+		Geography.Regions.Europe.Europe*             0.92
+		Geography.Regions.Europe.Northern Europe     0.915
+		Geography.Regions.Europe.Southern Europe     0.934
+		Geography.Regions.Europe.Western Europe      0.938
+		Geography.Regions.Oceania                    0.911
+		History and Society.Business and economics   0.968
+		History and Society.Education                0.972
+		History and Society.History                  0.949
+		History and Society.Military and warfare     0.973
+		History and Society.Politics and government  0.957
+		History and Society.Society                  0.907
+		History and Society.Transportation           0.988
+		STEM.Biology                                 0.986
 		STEM.Chemistry                               0.988
-		STEM.Computing                               0.987
-		STEM.Earth and environment                   0.98
-		STEM.Engineering                             0.986
-		STEM.Libraries & Information                 0.974
-		STEM.Mathematics                             0.989
-		STEM.Medicine & Health                       0.981
-		STEM.Physics                                 0.988
-		STEM.STEM*                                   0.983
-		STEM.Space                                   0.994
-		STEM.Technology                              0.981
+		STEM.Computing                               0.986
+		STEM.Earth and environment                   0.976
+		STEM.Engineering                             0.983
+		STEM.Libraries & Information                 0.969
+		STEM.Mathematics                             0.986
+		STEM.Medicine & Health                       0.977
+		STEM.Physics                                 0.986
+		STEM.STEM*                                   0.981
+		STEM.Space                                   0.991
+		STEM.Technology                              0.976
 		-------------------------------------------  -----
-	pr_auc (micro=0.826, macro=0.684):
+	pr_auc (micro=0.7, macro=0.534):
```

Comment:
Substantial decrease in fitness.  We might need to increase the vocab size for viwiki.